README.md

```diff
@@ -91,7 +91,7 @@ $ ethersync share
 Peer connected: adfa90edd932732ddf242f24dc2dcd6156779e69966d432fcb3b9fe3ae9831ab
 ```
 
-In another directory (this can be on another computer!):
+Another person, in a separate directory (also works on the same computer):
```

Comment:
This is still a bit confusing. So we have three things to convey:
- The usual case is to connect with another person on their computer
- It's also possible to connect with yourself on the same computer
- It has to be a different directory, of course different computer fulfills that automatically

How to bring this into one tiny sentence?
```suggestion
In a separate directory (usually another person & computer, but also works on the same computer):
```

Reply:
after discussing, I like the existing solution :)